Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -8,16 +8,34 @@ import UIKit
 internal final class CommentsViewController: UITableViewController {
   // MARK: - Properties
 
+  private lazy var commentComposer: CommentComposerView = {
```

Comment:
Can this raw value of height be referenced internally through an `enum` like with the other views?